app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -18,49 +18,81 @@ class GetUspsProofingResultsJob < ApplicationJob
 
   discard_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError
 
+  def enrollment_attributes(enrollment, enrollment_completed)
+    {
+      enrollment_code: enrollment.enrollment_code,
+      enrollment_id: enrollment.id,
+      minutes_since_last_status_check: enrollment.minutes_since_last_status_check,
+      minutes_since_last_status_update: enrollment.minutes_since_last_status_update,
+      minutes_to_completion: enrollment_completed ? enrollment.minutes_since_established : nil,
+    }
+  end
+
   def perform(_now)
     return true unless IdentityConfig.store.in_person_proofing_enabled
 
-    proofer = UspsInPersonProofing::Proofer.new
-
+    @enrollment_outcomes = {
```

Comment:
I was wondering if this could be a return value of `check_enrollments`, but I can see that the way we're incrementing it makes it hard to do that way. Not sure I have any suggestions to offer, but I'd have a soft worry it may be difficult to track / enforce consistency in how we're incrementing these values.

I think it'd be fine to trust our test coverage and consider revisiting later.

Reply:
I'm going to go ahead and leave it as-is for now